pom.xml

```diff
@@ -53,7 +53,7 @@
         <dep.drift.version>1.14</dep.drift.version>
         <dep.joda.version>2.10.3</dep.joda.version>
         <dep.tempto.version>166</dep.tempto.version>
-        <dep.testng.version>6.10</dep.testng.version>
+        <dep.testng.version>7.0.0</dep.testng.version>
```

Comment:
Maybe i shouldn't say that, but I'd prefer not to use version matching `\.0\.0$`

Reply:
It fails in so many ways.. ; /